File: FrontEnd/SecretOfTheDay/SecretOfTheDayGUI/Source/Model/Guess.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SecretOfTheDayGUI.Source.Model
{
    class Guess
    {
        Pineapple voter;
        Pineapple votee;

        public Guess(Pineapple voter , Pineapple votee)
        {
            this.voter = voter;
            this.votee = votee;
        }
    }
}

File: BackEnd/src/main/java/com/cegekaschool/secretguessinggame/model/guesses/Guess.java
package com.cegekaschool.secretguessinggame.model.guesses;

import com.cegekaschool.secretguessinggame.model.pineapples.Pineapple;

/**
 * Created by Thomas on 22/02/2017.
 */
public class Guess {

    private  Pineapple voter;
    private  Pineapple votee;

    private Guess(Pineapple voter, Pineapple votee) {
        this.voter = voter;
        this.votee = votee;
    }

    public static Guess createGuess(Pineapple voter, Pineapple votee) {

        return new Guess(voter, votee);
    }
}
